mne/commands/mne_watershed_bem.py

```diff
@@ -0,0 +1,166 @@
+"""
+
+    Create BEM surfaces using the watershed algorithm included with
+        FreeSurfer
+
+"""
+
+from __future__ import print_function
+import sys
+import os
+import shutil
+import mne
+from mne.utils import (verbose, logger)
+
+
+@verbose
+def mne_watershed_bem(subject=None, subjects_dir=None, overwrite=False,
+                      volume='T1', atlas=False, gcaatlas=False, preflood=None,
+                      verbose=None):
```

Comment:
It probably makes sense to put this function in `mne.bem` instead of here in commands. And call it something more in line with the rest of `mne-python` like `make_watershed_bem` (the `mne` prefix is not meaningful because it will already be `mne.whatever`). That way it can be called directly from Python in addition to being called from a shell, which is the only way to get to it currently.